src/game/base.ts

```diff
@@ -131,7 +126,10 @@ export class Game implements IGame {
         // Post render for positional adjustments to game.
         await this.currentRound.postRender(this);
 
-        if (this.currentRound.isTimedRound()) {
+        if (
+            this.currentRound.isTimedRound() &&
+            this.difficulty != GameDifficulty.Noob
+        ) {
```

Comment:
I'm wondering if keeping this logic in the `#isTimedRound` method would make more sense. What do you think about passing either the game instance, or the difficulty, into `#isTimedRound` and handling the logic there? `Round#postRender` already expects an instance of `Game`, and `Round#getTimeoutTime` expects the difficulty. Since those methods already expect these things, this would not be a new precedent.